analysis/staticAnalysisReport.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "Eval Static Analysis"
+author: "Chakshu Goyal"
+date: "07/06/2018"
+output:
+  html_document: default
+  pdf_document: default
+---
+
+## 
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(ggplot2)
+library(tidyverse)
+library(tools)
+```
+
+## Load the data
+Usage of the function eval in R-packages
+```{r}
+evaldf <- read_csv("evals.csv")
+```
+
+## Plot occurences of evals in packages
+```{r one}
```

Comment:
knit automatically names chunks - no need to say "one" - better no name or something which describes the tasks. I usually use no names - only the long running ones.

Reply:
Ahh okay! Acknowledged.